packages/pass-style/src/internal-types.js

```diff
@@ -17,11 +18,12 @@ export {};
  *
  * @typedef {object} PassStyleHelper
  * @property {PassStyle} styleName
- * @property {(candidate: any, check?: Checker) => boolean} canBeValid
- * If `canBeValid` returns true, then the candidate would
+ * @property {(candidate: any, reject: Rejector) => boolean} confirmCanBeValid
+ * If `confirmCanBeValid` returns true, then the candidate would
  * definitely not be valid for any of the other helpers.
  * `assertRestValid` still needs to be called to determine if it
- * actually is valid, but only after the `canBeValid` check has passed.
+ * actually is valid, but only after the `confirmCanBeValid` check has passed.
+ *
  * @property {(candidate: any,
  *             passStyleOfRecur: (val: any) => PassStyle
  *            ) => void} assertRestValid
```

Comment:
In a later PR, we should make this a confirm method as well, enabling us to test passsibility, or to call a "predicate" variant of `passStyleOf` that returns `undefined` rather than throwing for a non-passible argument.

No change suggested.

Reply:
Agreed.